app/controllers/Application.scala

```diff
@@ -154,7 +154,11 @@ object Application extends Controller {
   def npmPackageVersions(packageName: String) = Action.async {
     val packageVersionsFuture = Cache.getAs[Seq[String]](s"npm-versions-$packageName").fold {
       npm.info(packageName).map { json =>
-        val versions = (json \ "versions" \\ "version").map(_.as[String])
+        val versions = (json \ "versions" \\ "version")
+          .map(_.as[String])
+          .map(WebJarVersion(_))
+          .sorted(WebJarVersion.WebJarVersionOrdering.reverse)
+          .map(_.number)
```

Comment:
I think you can use the sort function directly. Know what I mean? We should probably refactor it out of the WebJar file though.